lib/cloud-metadata/index.js

```diff
@@ -171,10 +142,10 @@ class CloudMetadata {
  * Will output metadata object or error if no servers are reachable
  */
 function main (args) {
-  const cloudMetadata = new CloudMetadata({
-    _conf: {
-      cloudProvider: 'auto'
-    }
+  const cloudMetadata = new CloudMetadata('auto', {
```

Comment:
If you want a no-op logger, you can get one via:

```
const logging = require('../logging')
logging.createLogger('off')
```

See https://github.com/elastic/apm-agent-nodejs/blob/master/lib/logging.js#L62-L69

Reply:
✅ done